src/client/components/app/Transactions/WithdrawCreditTx.tsx

```diff
@@ -179,7 +203,7 @@ export const WithdrawCreditTx: FC<BorrowCreditProps> = (props) => {
         inputError={false}
         amount={targetAmount}
         onAmountChange={onAmountChange}
-        maxAmount={'0'}
+        maxAmount={getMaxWithdraw()}
```

Comment:
Is this simply a UI update? What happens if the `getMaxWithdraw()` fn returns without a value as dictated by:
```
    if (!selectedPosition) {
      setErrors([...errors, 'no selected position']);
      return;
    }
```